package/CHANGELOG

```diff
@@ -78,6 +78,7 @@ Enhancements
    DOI 10.1021/acs.jpcb.7b11988. (Issue #2039, PR #4524)
 
 Changes
+ * only use distopia < 0.3.0 due to API changes (Issue #4739)
```

Comment:
Not sure if we would normally make a CHANGELOG entry but in this case I would recommend to do so because there is not other part in the user-visible code base that knows about this requirement. If someone installs distopia by themselves, they will see failures.

Reply:
Now they will (hopefully) a warning
```
../package/MDAnalysis/lib/_distopia.py:45
  ~/MDAnalysis/mdanalysis/package/MDAnalysis/lib/_distopia.py:45: RuntimeWarning: Install 'distopia>=0.2.0,<0.3.0' to be used with this release of MDAnalysis. Your installed version of distopia >=0.3.0 will NOT be used.
    warnings.warn("Install 'distopia>=0.2.0,<0.3.0' to be used with this release "
```